android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -78,16 +79,16 @@ public class BraveRewardsHelper implements LargeIconBridge.LargeIconCallback {
     private static final int FAVICON_TEXT_SIZE = 50; // dp
     private static final int FAVICON_FETCH_INTERVAL = 1000; // In milliseconds
     private static final int FAVICON_DESIRED_SIZE = 64; // px
-    private static LargeIconBridge sLargeIconBridge;
+    @Nullable private static LargeIconBridge sLargeIconBridge;
```

Comment:
shouldn't it be before type per https://chromium.googlesource.com/chromium/src/+/main/styleguide/java/nullaway.md?

Reply:
it should, thanks, fixing

Reply:
fixed at https://github.com/brave/brave-core/pull/28726/commits/731b6220b4be7dafe514d22f02e015075d9c2e09